napari/_app_model/constants/_commands.py

```diff
@@ -100,11 +100,11 @@ class CommandId(str, Enum):
     IMAGE_ORIENT_PLANE_NORMAL_ALONG_Y = "napari:image:orient_plane_normal_along_y"
     IMAGE_ORIENT_PLANE_NORMAL_ALONG_X = "napari:image:orient_plane_normal_along_x"
     IMAGE_ORIENT_PLANE_NORMAL_ALONG_VIEW_DIRECTION = "napari:image:orient_plane_normal_along_view_direction"
-    IMAGE_HOLD_TO_PAN_ZOOM = "napari:image:hold_to_pan_zoom"
+    IMAGE_HOLD_TO_PAN_ZOOM = "napari:image:image_hold_to_pan_zoom"
```

Comment:
would it be possible to make it so that in the action manager shim it just prepends `[layer name]_` to the function instead of changing the app-model action id?

Reply:
Yes, definitely - the only issue there is it would change many of the entries in `utils/shortcuts.py` to match. Not hard just a little tedious.

Reply:
ok, I think I'll merge it as is and make the changes myself then